api/src/main/java/uk/gov/hmcts/PaymentApiApplication.java

```diff
@@ -58,7 +68,10 @@ public CacheManager cacheManager() {
                     .build()),
             new CaffeineCache("sites", Caffeine.newBuilder()
                 .expireAfterWrite(48, TimeUnit.HOURS)
-                .build())
+                .build()),
+        new CaffeineCache("userInfoCache", Caffeine.newBuilder()
```

Comment:
Please remove this cache for userinfo and check if it works

Reply:
Tried removing the **userInfoCache** but while authenticating the request we are not able to authorize the user.